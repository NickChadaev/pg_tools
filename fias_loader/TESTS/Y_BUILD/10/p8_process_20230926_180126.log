2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_10 postgres /home/n.chadaev@abrr.local/Y_BUILD/10/stage_gar_c_10.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Карелия Респ, id_region: 11, Execute: /home/n.chadaev@abrr.local/Y_BUILD/10/stage_gar_c_10.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_REESTR_OBJECTS_20230925_d3e1b612-700b-4fdb-af98-b8899e4248c2.XML
[18:06:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_ADDR_OBJ_20230925_23358851-9313-4e8c-a7dc-dde6b1b0af43.XML
[18:06:29] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_HOUSES_20230925_007f90be-8b45-4591-b76e-a9412058316d.XML
[18:09:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_STEADS_20230925_9ca71f27-dc1e-452f-a813-7f6d9e506e58.XML
[18:09:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_APARTMENTS_20230925_dbea5d71-396c-477b-9fd8-f91c10f45cc1.XML
[18:12:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_ROOMS_20230925_4fc3382d-23a8-4ab4-9db6-b830887819ba.XML
[18:12:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_CARPLACES_20230925_574bfa05-6923-4383-82af-f8708d8cd5b6.XML
[18:12:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_ADM_HIERARCHY_20230925_1710f8fe-5c19-4cf9-8a2d-5997bf0a4128.XML
[18:18:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_MUN_HIERARCHY_20230925_f49f989f-3f5f-4f87-942b-d2cb967610e9.XML
[18:23:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_CHANGE_HISTORY_20230925_7e15d5e9-6837-4d5f-aea8-e74c76e33147.XML
[18:35:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_ADDR_OBJ_DIVISION_20230925_a1a6a22c-10ac-4a9b-b8d3-329d939ede44.XML
[18:35:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_ADDR_OBJ_PARAMS_20230925_729a7fab-b750-494d-a612-2117f92af703.XML
[18:36:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_STEADS_PARAMS_20230925_54860fbd-7c25-4dc9-bab9-99f319453874.XML
[18:39:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_HOUSES_PARAMS_20230925_17dedc07-12d2-422e-8bab-f2700bd9ef6b.XML
[18:56:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_APARTMENTS_PARAMS_20230925_6fff556c-159f-479f-97c3-ef4819fa68f4.XML
[19:05:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_ROOMS_PARAMS_20230925_aced7701-eea0-4ed5-bcd0-2f0fd22785bc.XML
[19:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_CARPLACES_PARAMS_20230925_32a6e1a2-6ac8-428c-a943-86cc4b935f75.XML
[19:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/10/AS_NORMATIVE_DOCS_20230925_5e16c728-18af-47a9-aa62-0d36b37f6cf0.XML
[19:06:07] ...  -- Data finish
[19:06:07] ...  -- Set LOGGED
[19:06:19] ... Stop process
--------------------------------------------------------
