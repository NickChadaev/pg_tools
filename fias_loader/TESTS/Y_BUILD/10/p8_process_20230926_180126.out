 save_gar_version 
------------------
               70
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                70
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 404424
gar_fias.as_addr_obj: 12107
gar_fias.as_houses: 191050
gar_fias.as_steads: 15261
gar_fias.as_apartments: 286658
gar_fias.as_rooms: 2930
gar_fias.as_carplaces: 685
gar_fias.as_adm_hierarchy: 429245
gar_fias.as_mun_hierarchy: 443678
gar_fias.as_change_history: 968985
gar_fias.as_addr_obj_division: 65
gar_fias.as_addr_obj_params: 139088
gar_fias.as_steads_params: 185832
gar_fias.as_houses_params: 1410048
gar_fias.as_apartments_params: 657689
gar_fias.as_rooms_params: 3949
gar_fias.as_carplaces_params: 13
gar_fias.as_normative_docs: 65188
